src/cascade/input_data/db/__init__.py

```diff
@@ -1,3 +1,13 @@
+try:
+    import db_queries
+except ImportError:
+
+    class DummyDBQueries:
+        def __getattr__(self, name):
+            raise ImportError(f"Required package db_queries not found")
+
+    db_queries = DummyDBQueries()
+
 AGE_GROUP_SET_ID = 12
 
 GBD_ROUND_ID = 5
```

Comment:
I know this isn't the code under question, but these need to be somewhere else. I'd prefer to have them all the way up in the admin interface on EpiViz.

Reply:
Yeah. We need a parameters section. And I want to work to make it small. Even if they are hidden parameters in the EpiViz interface, that would make sense.